src/commands/lambda/flare.ts

```diff
@@ -441,11 +441,16 @@ export const writeFile = (filePath: string, data: string) => {
  * @returns the CSV string
  */
 export const convertToCSV = (logEvents: OutputLogEvent[]) => {
-  const rows = [['timestamp', 'message']]
+  const rows = [['timestamp', 'datetime', 'message']]
```

Comment:
Do we want to call the new column `datetime` or is there a better name?

Reply:
That's good